changelog.md

```diff
@@ -21,6 +21,7 @@
 
 ### Fixes
 
+* [1535](https://github.com/zeta-chain/node/issues/1535) - Avoid voting on wrong ballots due to false blockNumber in EVM tx receipt
```

Comment:
@ws4charlie can we add the change above, under `Unreleased`?

Reply:
[commit](e06624e0943aeb25493e87b245327ef3efa3ba81)